cobalt/media/base/metrics_provider.cc

```diff
@@ -121,21 +121,38 @@ void MediaMetricsProvider::EndTrackingAction(WebMediaPlayerAction action) {
   tracked_actions_start_times_.erase(action);
 }
 
-bool MediaMetricsProvider::IsActionCurrentlyTracked(
-    WebMediaPlayerAction action) {
+bool MediaMetricsProvider::IsActionCurrentlyTracked(MediaAction action) {
   ScopedLock scoped_lock(mutex_);
   return tracked_actions_start_times_.find(action) !=
          tracked_actions_start_times_.end();
 }
 
 void MediaMetricsProvider::ReportActionLatencyUMA(
-    WebMediaPlayerAction action, const base::TimeDelta& action_duration) {
+    MediaAction action, const base::TimeDelta& action_duration) {
   switch (action) {
-    case WebMediaPlayerAction::SEEK:
+    case MediaAction::WEBMEDIAPLAYER_SEEK:
       UMA_HISTOGRAM_TIMES("Cobalt.Media.WebMediaPlayer.Seek.Timing",
                           action_duration);
       break;
-    case WebMediaPlayerAction::UNKNOWN_ACTION:
+    case MediaAction::SBPLAYER_CREATE:
+      UMA_HISTOGRAM_CUSTOM_MICROSECONDS_TIMES(
+          "Cobalt.Media.SbPlayer.Create.Timing", action_duration,
+          base::TimeDelta::FromMicroseconds(100),
+          base::TimeDelta::FromMilliseconds(5000), 50);
+      break;
+    case MediaAction::SBPLAYER_CREATE_URL_PLAYER:
+      UMA_HISTOGRAM_CUSTOM_MICROSECONDS_TIMES(
+          "Cobalt.Media.SbPlayer.CreateUrlPlayer.Timing", action_duration,
+          base::TimeDelta::FromMicroseconds(100),
+          base::TimeDelta::FromMilliseconds(5000), 50);
+      break;
+    case MediaAction::SBPLAYER_DESTROY:
+      UMA_HISTOGRAM_CUSTOM_MICROSECONDS_TIMES(
+          "Cobalt.Media.SbPlayer.Destroy.Timing", action_duration,
+          base::TimeDelta::FromMicroseconds(100),
```

Comment:
So 100μs to 100000000μs is an enormous range to be divided into 50 buckets. Do you need such a wide range? As-is, your buckets have the potential to be quite coarse and possibly not useful. I'd narrow this as much as you can. Same with the above.

Reply:
woof! my mistake, they should all be in microseconds. Now corrected.

Reply:
Adjusted range. On Linux i was seeing ~500μs for Create and ~15000μs for destroy. 
I've adjusted range to 100 - 1500μs for create, and 500 - 40000μs for destroy.